paasta_tools/cli/cmds/logs.py

```diff
@@ -100,9 +103,37 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+
+    status_parser.add_argument(
+        '-a', '--ago', dest='time_ago',
+        help='How long ago to get logs from, uses friendly durations like 32m or 1w. Defaults to 30 minutes',
+    )
+    status_parser.add_argument(
+        '-t', '--time', dest='time',
```

Comment:
So the reason I decided not to have a from and to is because I feel like it is much more convenient for end users. I agree, it is a bit confusing at first glance but figuring out the timestamps for `from` and `to` yourself is so much of a hassle. 

In most cases people will be wanting logs relative to the current time, so for example they get a report that their service is misbehaving, the report says the problem was encountered like 5 minutes ago. Well then it's simple enough to do `paasta logs --ago 10m` and get all the logs from 10 minutes ago. However if you had to manually figure out the timestamps you'd have like `paasta logs --from 2016-06-06T16:25:30+00:00 --to 2016-06-06T16:35:30+00:00`